pkg/util/httputil/noindex.go

```diff
@@ -0,0 +1,12 @@
+package httputil
+
+import (
+	"net/http"
+)
+
+func NoIndex(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("X-Robots-Tag", "noindex")
```

Comment:
`noindex, nofollow`

Reply:
It seems `none` is better.
https://developers.google.com/search/docs/crawling-indexing/robots-meta-tag#nofollow

Reply:
Not sure if Bing knows `none`.

Reply:
https://www.bing.com/webmasters/help/which-robots-metatags-does-bing-support-5198d240
Ok maybe we use noindex, nofollow to be safe